.github/workflows/test.yml

```diff
@@ -74,7 +74,10 @@ jobs:
             exit 1
         fi
 
-    - name: Test
+    - name: Test js library
+      run: sudo -u testbot bash -lc 'make test-web'
+
```

Comment:
Forgot to mention: we should send the Playwright test results to a GH Actions artifact, like we do in app and RDK. This lets us view traces locally and diagnose test failures (and flakiness) more easily